web/src/engine/common/web-utils/src/tests/stringHandling.js

```diff
@@ -1,342 +1,342 @@
+      assert.equal(KMWString.indexOf(banana_nation, "na", 8), -1);
+      assert.equal(KMWString.indexOf(banana_nation, "na", 1000), -1);
+    });
+
+    it('lastIndexOf', () => {
       const banana_nation = "banana nation";
-      assert.equal(banana_nation._kmwLastIndexOf("XYZ"), -1);
-      assert.equal(banana_nation._kmwLastIndexOf("na"), 7);
-      assert.equal(banana_nation._kmwLastIndexOf("na", -1), -1);
-      assert.equal(banana_nation._kmwLastIndexOf("na", 0), -1);
-      assert.equal(banana_nation._kmwLastIndexOf("na", 2), 2);
-      assert.equal(banana_nation._kmwLastIndexOf("na", 3), 2);
-      assert.equal(banana_nation._kmwLastIndexOf("na", 5), 4);
-      assert.equal(banana_nation._kmwLastIndexOf("na", 8), 7);
-      assert.equal(banana_nation._kmwLastIndexOf("na", 1000), 7);
-    });
-
-    it('_kmwLength', () => {
-      assert.equal(""._kmwLength(), 0);
-      assert.equal("string"._kmwLength(), 6);
+      assert.equal(KMWString.lastIndexOf(banana_nation, "XYZ"), -1);
+      assert.equal(KMWString.lastIndexOf(banana_nation, "na"), 7);
+      assert.equal(KMWString.lastIndexOf(banana_nation, "na", -1), -1);
+      assert.equal(KMWString.lastIndexOf(banana_nation, "na", 0), -1);
+      assert.equal(KMWString.lastIndexOf(banana_nation, "na", 2), 2);
+      assert.equal(KMWString.lastIndexOf(banana_nation, "na", 3), 2);
+      assert.equal(KMWString.lastIndexOf(banana_nation, "na", 5), 4);
+      assert.equal(KMWString.lastIndexOf(banana_nation, "na", 8), 7);
+      assert.equal(KMWString.lastIndexOf(banana_nation, "na", 1000), 7);
+    });
+
+    it('length', () => {
+      assert.equal(KMWString.length(""), 0);
+      assert.equal(KMWString.length("string"), 6);
       // From the current MDN example:
-      assert.equal("Life, the universe and everything. Answer:"._kmwLength(), 42);
+      assert.equal(KMWString.length("Life, the universe and everything. Answer:"), 42);
     });
 
-    it('_kmwSlice', () => {
+    it('slice', () => {
       // From https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/slice
       const str = "The quick brown fox jumps over the lazy dog.";
 
       // FAILS:  does not handle `undefined` length like the real .slice() method.
-      // assert.equal(str._kmwSlice(31), "the lazy dog.");
-      assert.equal(str._kmwSlice(4, 19), "quick brown fox");
+      // assert.equal(KMWString.slice(str, 31), "the lazy dog.");
+      assert.equal(KMWString.slice(str, 4, 19), "quick brown fox");
       // FAILS:  does not handle `undefined` length like the real .slice() method.
-      // assert.equal(str._kmwSlice(-4), "dog.");
-      assert.equal(str._kmwSlice(-9, -5), "lazy");
+      // assert.equal(KMWString.slice(str, -4), "dog.");
+      assert.equal(KMWString.slice(str, -9, -5), "lazy");
       // FAILS:  does not handle `undefined` start like the real .slice() method.
-      // assert.equal(str._kmwSlice(), str);
+      // assert.equal(KMWString.slice(str), str);
```

Comment:
Should enable and fix these tests.

Reply:
That will not be done in this PR.  We don't mix refactors and bugfixes when we can help it.

Also, we don't actually use `slice` in our code, so it'd be a low-priority fix.

Reply:
#13784